.rubocop.yml

```diff
@@ -36,3 +36,11 @@ RSpec/ExampleLength:
 RSpec/DescribeClass:
   Exclude:
     - spec/project/**/*.rb
+
+# FIXME
+Style/FormatStringToken:
```

Comment:
I can fix our codebase regarding this ;)

Reply:
That would be great ❤️

Reply:
@bquorning Done, please review